api/problem_details.py

```diff
@@ -191,13 +191,6 @@
       _("Could not fulfill loan."),
 )
 
-NO_ACCEPTABLE_FORMAT = pd(
-    "http://library-simplified.org/terms/problem/no-acceptable-format",
-    502,
-    _("No acceptable format."),
-    _("This book is not available in the requested format."),
-)
-
```

Comment:
This change ended up being unrelated, but I saw that NO_ACCEPTABLE_FORMAT was defined twice in this file.